blueocean-dashboard/src/main/js/creation/bitbucket/cloud/BbCloudFlowManager.js

```diff
@@ -362,21 +375,45 @@ export default class BbCloudFlowManager extends FlowManager {
             this._logEvent(event);
         }
 
-        if (event.job_multibranch_indexing_status === 'INDEXING' && event.job_run_status === 'QUEUED') {
-            this.queuedIndexAllocations++;
-        }
-
-        if (event.job_multibranch_indexing_result) {
-            this.queuedIndexAllocations--;
+        if (event.jenkins_object_type === 'org.jenkinsci.plugins.workflow.job.WorkflowRun') {
+            if (event.job_run_status === 'ALLOCATED' || event.job_run_status === 'RUNNING') {
+                this._checkForBranchCreation(event.blueocean_job_pipeline_name, false, ({ hasError, pipeline }) => {
+                    if (!hasError) {
+                        this._finishListening(STATE.PENDING_CREATION_EVENTS);
```

Comment:
Should this `STEP_COMPLETE_SUCCESS`? I assume this code block indicates a run has started for one of the pipeline's branches, so we know it must have a Jenkinsfile?